### PR TITLE
add READMEs with demo tutorials

### PR DESCRIPTION
**Pull request by Cerchie**
adding a few new demos for the confluent developer demo page.

### CODE REVIEW

File: confluent-connector-github-demo/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+For 'GitHub Endpoint', enter `https://api.github.com`.
+
+For 'GitHub Access Token', enter the classic personal token you created earlier. You do not need to preface it with 'Bearer'. 
+
+Next, add configuration details.
+
+<img width="955" alt="connection details" src="https://user-images.githubusercontent.com/54046179/222235864-4324240b-d82f-4ba1-adfd-e52f92b22573.png">
+
+> Note: copy and paste this value into the 'Topic Name Pattern' field:  `github-${resourceName}` 
+
```
Comment (davetroiano) [inline confluent-connector-github-demo/README.md:52]: instruct user to pick an output record value format. JSON?

File: confluent-xml-demo/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+After you have those values, you can configure your `kcat` utility! [Use the instructions written by Kris Jenkins](http://blog.jenkster.com/2022/10/setting-up-kcat-config.html), and the values you've just generated. 
+
+## Step 3: Pipe your data into the topic! 
+
+Run the command:
+
+```bash
+curl --show-error --silent https://tfl.gov.uk/tfl/syndication/feeds/cycle-hire/livecyclehireupdates.xml | \
+    xq -c '.stations.station[] + {lastUpdate: .stations."@lastUpdate"}' | \
+    kcat -t livecyclehireupdates_01 -P
```
Comment (davetroiano) [inline confluent-xml-demo/README.md:64]: add the other `kcat` params `-X security.protocol=SASL_SSL -X sasl.mechanism=PLAIN -b BROKER_ENDPOINT -X sasl.username=API_KEY -X sasl.password=API_SECRET`

File: confluent-ais-demo/README.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+
+View the [instructions](https://stedolan.github.io/jq/download/) for installing `jq` depending on your operating system. 
+
+For `gpsd`, `brew install gpsd` for MacOS,
+
+or 
+
+`apt-get install gpsd-clients` for Linux.
+
+unfortunately, installation is [not recommended for WSL2](https://gpsd.gitlab.io/gpsd/installation.html) and linux distribution is recommended.
```
[the remark this answers is not in the document]
Comment (Cerchie) [reply confluent-ais-demo/README.md]: done

File: confluent-connector-github-demo/README.md (hunk 1 above)
Comment (Cerchie) [reply confluent-connector-github-demo/README.md:52]: done

File: confluent-xml-demo/README.md (hunk 2 above)
Comment (Cerchie) [reply confluent-xml-demo/README.md]: done

File: confluent-connector-github-demo/README.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
+# Confluent Connector Github Demo 
+
+In this demo, you'll learn how to set up the Confluent GitHub Connector, and then how to generate your own data with commit events that surface to the Confluent interface! 
+
+Let's get started. 
+
+## Prerequisites
+
+1. A [Confluent Cloud](https://rebrand.ly/f0kvol5) account. 
+2. A [GitHub account](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account). You'll also need a [classic personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-personal-access-token-classic) for accessing the GitHub REST API.  To test your token, run this command in your terminal:
```
[the remark this answers is not in the document]
Comment (Cerchie) [reply confluent-connector-github-demo/README.md]: done... & thank you for the review I appreciate it!

**Pull request merged**